.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Report a bug
+description: Found something? Thank you for telling us!
+labels: ["bug"]
+---
+
+<!-- Please provide the following information to help us diagnose the issue, removing any sections that are not applicable. Thank you for your help! -->
+
+## Problem statement
+
+<!-- Describe the problem you are experiencing: -->
+
+## Steps to reproduce
+
+<!-- List the steps to reproduce the behavior: -->
+
+1. Go to '...'
+2. Click on '...'
+3. Scroll down to '...'
+
+**Expected behavior:** <!-- What should have happened? -->
+
+**Actual behavior:** <!-- What actually happened? -->
+
+## System Details
+
+<!-- Click Copy System Info under Help and Feedback in the sidebar to copy system info to your clipboard, and paste it here: -->
+
+## Publishing information
+
+- **Content type:** <!-- e.g., FastAPI app, Shiny app, Quarto document -->
+
+<!--
+If applicable: find this in Connect on System → Info tab → under System Info. See https://docs.posit.co/connect/admin/server-management/#determine-version-installed.
+
+**Connect version:** e.g., 2025.09.0
+-->
+
+<!--
+If applicable: find this in Connect Cloud at https://connect.posit.cloud/my-account.
+
+**Connect Cloud account name:** e.g., my-account
+-->
+
+## Publisher output and logs
+
+<!-- Publisher output helps us to identify the problem, find it in the "Output" window → "Posit Publisher Deployment" option (see https://github.com/posit-dev/publisher/blob/main/docs/troubleshooting.md#publisher-output). Paste Publisher output here: -->
```

Comment:
Not something we should solve here, but `Posit Publisher Deployment` should be cleaned up (I swear I had made an issue for this, but #3168 exists now!)